config/monitoring.sensor_config.ocha_google_tag.yml

```diff
@@ -0,0 +1,18 @@
+uuid: null
```

Comment:
This should be the gtm_barebones sensor

Reply:
See https://github.com/UN-OCHA/ocha_monitoring/pull/4